src/cc_catalog_airflow/dags/provider_api_scripts/test_finnish_museums.py

```diff
@@ -0,0 +1,125 @@
+import os
+import json
+import logging
+from unittest.mock import patch
+import finnish_museums as fm
+
+
+RESOURCES = os.path.join(
+    os.path.abspath(os.path.dirname(__file__)), "tests/resources/finnishmuseums"
+)
+
+logging.basicConfig(
+    format="%(asctime)s - %(name)s - %(levelname)s:  %(message)s", level=logging.DEBUG
```

Comment:
Please limit these lines to 79 chars.  While we allow long lines in test files, that exception is intended to handle long strings.  In this case, that exception doesn't apply.